backend/services/grading_embedding_service.py

```
@@ -0,0 +1,388 @@
+        db = SessionLocal()
+        
+        try:
+            material = db.query(GradingMaterial).filter(
+                GradingMaterial.id == material_id
+            ).first()
+            
+            if material:
+                material.processing_status = status
+                if log_data:
+                    material.processing_log = log_data
+                material.updated_at = datetime.utcnow()
```

Ruff (0.14.1)

`datetime.datetime.utcnow()` used

(DTZ003)